js/views/search/SearchProviders.js

```diff
@@ -83,7 +83,10 @@ export default class extends BaseView {
         const view = this.createProviderView(provider);
         if (view) {
           this.providerViews.push(view);
-          view.render().$el.appendTo(providerFrag);
+          if (provider.get(this.options.urlType)) {
```

Comment:
Could we do something with urlType in the constructor? Either default it or throw an exception if it's not provided? It's use multiple times in this view and, unless a dev digs through the code, they'd have no idea they should provide it.